docs/book/releases/v2.2.0.md

```diff
@@ -0,0 +1,72 @@
+<!-- markdownlint-disable MD034 -->
+# vSphere CSI Driver - v2.2.0 release
+
+## New Feature
+
+- Online Volume Expansion. Refer [Volume Expansion documentation](../features/volume_expansion.md) for detail.
+
+## Notable Changes
+
+- Datastore privilege check to filter out shared datastores which do not have `Datastore.FileManagement` privilege. Refer notes provided for `CNS-DATASTORE` role at [vSphere Roles and Privileges](https://vsphere-csi-driver.sigs.k8s.io/driver-deployment/prerequisites.html#vsphere-roles-and-privileges-)
+- Telemetry collection support using `cluster-distribution` field in the vSphere Config Secret. Refer notes provided for `cluster-distribution` in [vSphere configuration secret](https://vsphere-csi-driver.sigs.k8s.io/driver-deployment/installation.html#vsphere-configuration-file-for-block-volumes-).
+- Bug fixes (Issues #1, 2, 6, 7, and 8 mentioned in the known issues section of [v2.1.1](v2.1.1.md) are fixed in `v2.2.0` release)
+  - Fixed recreating static file share volume with the same PV name.
+  - Fixed metadata syncer to disallow deleting volume physically from the datastore when PV is deleted by user.
+  - Fixed VCP to CSI Migration to support datastore path in the volume handle. Refer [issue-628](https://github.com/kubernetes-sigs/vsphere-csi-driver/issues/628#issuecomment-771258857) for more detail.
+  - Fixed CSI full sync not registering volumes and was failing with `Duplicated entity for each entity type in one cluster`.
+  - Fixed volume provisioning when empty datacenter is present in the vCenter inventory.
+
+## Deployment files
+
+- https://github.com/kubernetes-sigs/vsphere-csi-driver/tree/master/manifests/v2.2.0
+
+## Kubernetes Release
+
+- Minimum: 1.18
+- Maximum: 1.20
+
+## Supported sidecar containers versions
+
+- csi-provisioner - v2.1.0
+- csi-attacher - v3.1.0
+- csi-resizer - v1.1.0
+- livenessprob - v2.2.0
+- csi-node-driver-registrar - v2.1.0
+
+## Known Issues
+
+### vSphere CSI Driver issues
```

Comment:
Can you add the following known issue to this list:
```
Volume resize operation fails with `A general system error occurred: Failed to lock the file: api = DiskLib_Grow` error
Issue: For volume expansion to work, the VC and all the ESX nodes of the cluster need to be on the supported versions of the feature i.e version 7.0 and above. 
Workaround: vSphere CSI driver supports offline expansion from vSphere 7.0 and online expansion from vSphere 7.0u2 onwards. Make sure all the ESX nodes present in the cluster are on the same version as the vCenter.
```

Reply:
This is feature limiation.
I am adding the following note at https://deploy-preview-769--kubernetes-sigs-vsphere-csi-driver.netlify.app/features/volume_expansion.html

> Make sure all the ESXi hosts of the cluster are on the same version as the vCenter.
For volume expansion to work, the vCenter and all the ESX hosts of the cluster need to be on the supported versions of the feature i.e version 7.0 and above, otherwise volume resize operation fails with `A general system error occurred: Failed to lock the file: api = DiskLib_Grow` error.

Reply:
Can we mention it as a known issue instead of making it a note in feature page? This will help the customers to diagnose easily by checking the known issue page and we plan to remedy this behavior in CSI v2.2.1, so we can mark it as fixed in that release too.

Reply:
this is also listed as a limitation that vCenter and ESX have to be on the same version, other newer features added in the vCenter will not work as expected.

Refer - https://deploy-preview-769--kubernetes-sigs-vsphere-csi-driver.netlify.app/compatiblity_matrix.html